docs/apm/using-the-apm-ui.asciidoc

```diff
@@ -2,90 +2,256 @@
 [[apm-ui]]
```

Comment:
Will probably break this out into multiple `asciidoc` files, but wanted to leave it in the original file for now so you can see the diff.